src/GraphQLEditor/ReduxGqlEditor.js

```diff
@@ -1,11 +1,15 @@
 import { connect } from 'react-redux';
-
+import querystring from 'querystring';
```

Comment:
I think it's ``` import queryString from 'query-string';```
And using `queryString` instead of `querystring` won't need to replace the prepend question mark just FYI